src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c

```diff
@@ -352,14 +352,11 @@ void func_80966F74(EnWeatherTag* this, PlayState* play) {
     }
 }
 
-// WEATHERTAG_TYPE_UNK4: no visible effect, what does it doooo??
-// used in south clock town??? romani ranch, clock tower rooftop woodfall..? stt
-// all of them have shorter distances though, like 0xA and 0x6, so their locations are important
 void func_80966FEC(EnWeatherTag* this, PlayState* play) {
-    // weirdly, not the same as the other param lookup used in the rest of the file, which is float
-    s32 distance = WEATHER_TAG_RANGE100INT(&this->actor);
-    if (distance > 0) {
-        gSkyboxNumStars = distance;
+    s32 numStars = WEATHER_TAG_GET_SKYBOX_NUM_STARS(&this->actor);
+
+    if (numStars > 0) {
+        gSkyboxNumStars = numStars;
```

Comment:
Part of the macro is to multiply the value by 100, so a param of `0xA` would actually be 100 starts. Do you think the `* 100` should be moved outside the macro?

Reply:
Oh I didn't see the macro... this is one of the things that used to bother me about macros, it moves implementation to the header instead of leaving it in the C file...

You _could_ leave it out of the macro, but if it's always times by the same value it makes sense to include in the macro.